gui/src/app/SamplerOutputView/SamplerOutputView.tsx

```diff
@@ -14,7 +14,7 @@ import TracePlotsView from "./TracePlotsView";
 type SamplerOutputViewProps = {
   width: number;
   height: number;
-  sampler: StanSampler;
+  sampler: StanSampler | undefined;
```

Comment:
This seems like an unexpected change for this PR -- what necessitates it?

Reply:
It's because we now need the output view to exist even when the stan program has not been compiled. It's one of the three tabs.